src/openfermion/ops/_binary_operator.py

```diff
@@ -0,0 +1,494 @@
+# symbolic binary class for decoder definitions (arxiv 1712.07067)
```

Comment:
Please copy the Apache 2 license header that is in the rest of the OpenFermion files. This is a fine one-line description but it should go under the header and should be commented
`"""Like this."""`
rather than
`# like this.`